mysql/Dockerfile

```diff
@@ -1,19 +1,47 @@
 FROM fedora
-MAINTAINER http://fedoraproject.org/wiki/Cloud
 
-RUN dnf -y update && dnf clean all
-RUN dnf -y install community-mysql-server community-mysql pwgen supervisor bash-completion psmisc net-tools && dnf clean all
+# MySQL image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql/data - Datastore for MySQL
+# Environment:
+#  * $MYSQL_USER - Database user name
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+#  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
 
-ADD ./start.sh /start.sh
-ADD ./config_mysql.sh /config_mysql.sh
-ADD ./supervisord.conf /etc/supervisord.conf
+MAINTAINER http://fedoraproject.org/wiki/Cloud
 
-# RUN echo %sudo	ALL=NOPASSWD: ALL >> /etc/sudoers
+ENV MYSQL_VERSION=10.0 \
+    HOME=/var/lib/mysql
 
-RUN chmod 755 /start.sh
-RUN chmod 755 /config_mysql.sh
-RUN /config_mysql.sh
+LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database server" \
+      io.k8s.display-name="MySQL 5.6" \
```

Comment:
I don't think 5.6 is correct either, because unlike with the SCL, you're not ensuring the version. I would drop the version numbers. These dockerfiles aren't for a specific version; they're for whatever is in the Fedora package. Maybe you'd want to write it as "Fedora MySQL" or "Fedora 23 MySQL" instead, I dunno.